packages/database-manager/package.json

```diff
@@ -43,7 +41,7 @@
   "author": "Elies Lou (Aetherall)",
   "license": "MIT",
   "devDependencies": {
-    "@accounts/server": "^0.1.0-beta.3",
+    "@accounts/types": "^0.1.0-beta.10",
```

Comment:
@accounts/types should be inside dependencies

Reply:
Okay It's fixed now